tests/property/test_input_validation.py

```
@@ -60,6 +60,10 @@ def _assert_only_tool_error(exc: BaseException) -> None:
     )
 
 
+def _make_tool():
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_make_tool`

(ANN202)